src/consensus/params.rs

```diff
@@ -46,7 +46,7 @@ const MAX_BITS_REGTEST: Uint256 = Uint256([
     0x0000000000000000u64,
     0x0000000000000000u64,
     0x0000000000000000u64,
-    0x7fffff0000000000u64,
+    0x00ffffffffffffffu64,
```

Comment:
This is the number in `SigNetParams`
`00000377ae000000000000000000000000000000000000000000000000000000`

Reply:
`00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff` is RegTest

Reply:
Nevermind, looking at the wrong code.